### PR TITLE
add policy AllowLocalSystemToUseComputerIdentity

### PR DESCRIPTION
**Pull request by yogeshasalkar**
The **Network security: Allow Local System to use computer identity for NTLM** is missing from the documentation. but can be configured via Intune using the OMA-URI 
**./Device/Vendor/MSFT/Policy/Config/LocalPoliciesSecurityOptions/NetworkSecurity_AllowLocalSystemToUseComputerIdentityForNTLM**

Thanks

### CODE REVIEW

**Comment by yogeshasalkar** [comment]
Hi @JohanFreelancer9 , could you please have a look?

**Comment by e0i** [comment]
@yogesha @illfated 

Please mention me, instead of Johan for any of the upcoming PRs, thank you.

**Comment by yogeshasalkar** [comment]
Thanks @e0i  for letting me know

File: windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md [hunk 1]
```diff
@@ -2166,6 +2169,73 @@ GP Info:
...
+Network security: Allow Local System to use computer identity for NTLM.
+
+When services connect to devices that are running versions of the Windows operating system earlier than Windows Vista or Windows Server 2008, services that run as Local System and use SPNEGO (Negotiate) that revert to NTLM will authenticate anonymously. In Windows Server 2008 R2 and Windows 7 and later, if a service connects to a computer running Windows Server 2008 or Windows Vista, the system service uses the computer identity.
+
+When a service connects with the device identity, signing and encryption are supported to provide data protection. (When a service connects anonymously, a system-generated session key is created, which provides no protection, but it allows applications to sign and encrypt data without errors. Anonymous authentication uses a NULL session, which is a session with a server in which no user authentication is performed; and therefore, anonymous access is allowed.)
+
+<!--/Description-->
+<!--RegistryMapped-->
+GP Info:  
+-   GP English name: *Network security: Allow Local System to use computer identity for NTLM.*
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md:2225]:
```suggestion
-   GP English name: *Network security: Allow Local System to use computer identity for NTLM*
```

**Comment by ManikaDhiman** [comment]
Policy owner approved these updates. I will create a new PR to have this policy setting added to the Policy CSP index page and What's new doc. Thanks!

**Pull request merged**